pallets/xcm-weight-trader/src/lib.rs

```diff
@@ -94,14 +110,20 @@ pub mod pallet {
 		type WeightInfo: WeightInfo;
 
 		/// Convert a weight value into deductible native balance.
-		type WeightToFee: WeightToFee<Balance = Self::Balance>;
```

Comment:
Was this giving any error?

Reply:
Removed it. Seems like it was left dangling from a previous implementation.